en/frames_multicopter/holybro_x500V2_pixhawk5x.md

```diff
@@ -0,0 +1,211 @@
+# Holybro X500 V2 + Pixhawk5x Build
```

Comment:
I'm not sure you need to mention the "Pixhawk5x Build" part, but if we really do please update to "Pixhawk 5X Build"